using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -89,7 +101,11 @@ This API synchronizes the provided customer identifier type key and value with t
          authentication: (ADBMobileVisitorAuthenticationState) authenticationState;
 ```
 
-#### **Examples**
+o  *identifierType* (String) containing identifier type, and the *identifier* (String) containing identifier value; should not be null or empty. If the identifierTtype and identifier do not have valid string values \(non null or empty\), the identifier is ignored by the Identity extension.
+
+o *authenticationState* (VisitorIDAuthenticationState) value indicating authentication state for the user contaisn one of the VisitorID.AuthenticationState values `ACPMobileVisitorAuthenticationStateAuthenticated`,`ACPMobileVisitorAuthenticationStateLoggedOut` and `ACPMobileVisitorAuthenticationStateUnknown`
```

Comment:
- Please remove "o"; as this is a sentence, and not a list of parameters, we don't need the bullet. 

- Please reword the text so that it reads as follows:
" The `authenticationState` (`VisitorIDAuthenticationState`) value indicates the authentication state for the user and contains one of the following VisitorID.AuthenticationState values:
  * `ACPMobileVisitorAuthenticationStateAuthenticated`
  * `ACPMobileVisitorAuthenticationStateLoggedOut`
  * `ACPMobileVisitorAuthenticationStateUnknown`